### PR TITLE
WAF-120: Make size option button smaller in smaller screens

### PR DESCRIPTION
**Pull request by yugandhar02**

### CODE REVIEW

File: src/components/AddToCartButton/AddToCartButton.tsx [hunk 1]
```diff
@@ -10,21 +10,24 @@ it.
...
   onClick: (e: any) => any;
 }
 export const AddToCartButton: FunctionComponent<AddToCartButtonProps> = ({
   onClick,
 }: AddToCartButtonProps) => {
+  const { screenSize } = useSensor();
+
   return (
-    <div className="ds-sdk-add-to-cart-button">
+    <div className={`ds-sdk-add-to-cart-button ${screenSize.desktop ? 'h-[100%] w-full' : 'h-[40px] w-[40px]'}`}>
```
Comment (rhudea) [inline src/components/AddToCartButton/AddToCartButton.tsx:24]: Why media queries couldn't be used here ?

Comment (yugandhar02) [reply src/components/AddToCartButton/AddToCartButton.tsx:24]: screenSize.desktop is a media query

Comment (rhudea) [reply src/components/AddToCartButton/AddToCartButton.tsx:24]: What I wanted to ask is  whether it can be done just in css ?

Comment (yugandhar02) [reply src/components/AddToCartButton/AddToCartButton.tsx:24]: the default tailwind config will serve our purpose though

Comment (yugandhar02) [reply src/components/AddToCartButton/AddToCartButton.tsx:24]: There is something wrong with nesting config
the media queries are not getting nested under .ds-widgets 
and hence the mobile classes are overriding the media queries classes

image

**Review by rhudea — APPROVED** [review]

**Pull request merged**